changelog.md

```diff
@@ -16,3 +16,7 @@
 - Rename `QueryProviderBuilder` to `QueryableBuilder`
 - Let `QueryableBuilder` return an `IQueryable<T>` directly
 - Provide an interface for `QueryableBuilder`
+
+## TBA
+
```

Comment:
Remove newline

Reply:
Addressed in https://github.com/messerli-informatik-ag/linq-to-rest/pull/27/commits/16f9f717fa3e4ec123a5107ee8fc422fa04e6ff8